tutorials/ios-chatbot/part-3/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: How to build a conversational app using Cloud Machine Learning APIs (Part 3 of 3)
+description: How to build a conversational app using Cloud Machine Learning APIs (Part 3 of 3).
+author: PokerChang
+tags: Cloud Functions, Dialogflow, API.AI, Webhooks, Localization, Chatbot, Machine Learning API, Transation, Vision, Speech
+date_published: 2017-01-02
+---
+
+In [part 1] and [part 2] of this series, we showed you how to build a conversational tour guide app with API.AI and Google Cloud Machine Learning APIs. In this final part, you'll learn how to extend this app to the Google Assistant-supported devices (Google Home, eligible Android phones and iPhones, and Android Wear). And we'll build this on top of the existing API.AI agent created in parts 1 and 2.
+
+[![The Google Assistant / Google Home Demo](http://img.youtube.com/vi/_x5rlkpZiyc/0.jpg)](https://youtu.be/_x5rlkpZiyc)
+
+### <a name="actions"></a>New Intents for Actions on Google
+
+In [part 1], we discussed the app's input and output context relationships. 
+
+
+![alt_text](conversational-app-p3-3.png "Contexts without the Assistant")
+
+
+The _where_ context requires the user to upload an image, which is not supported by the Google Assistant. We can modify the context relationship as below. 
+
+![alt_text](conversational-app-p3-8.png "Contexts with the Assistant")
+
+We will add three new intents, _hours-no-context_, _ticket-no-context_ and _map-no-context_. Each intent will set _location_ as the output context so that other intents can use the _location_ as an input parameter.
+
+![alt_text](conversational-app-p3-5.png "Contexts Screenshot")
+
+### Enable Actions on Google Integration
+
+Now we'll enable Actions on Google to support the Google Assistant.
+
+1. Open your API.AI console. Under the _Integrations_ Tab, turn on the _Actions on Google_ integration.
```

Comment:
In Part 2, I asked you to capitalize and bold the names of the tabs. You can use italics if you want, but just be consisten with it.